### PR TITLE
Accounts route

### PR DESCRIPTION
**Pull request by jkranking**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
 
 get 'test', to: 'test#jeff'
+
+get '/accounts', to: 'accounts#index'
+
+get '/accounts/:id', to: 'accounts#show', as: 'account'
```
Comment (Andrew-Max) [inline config/routes.rb:7]: I believe if we had used the resource syntax here, we would have gotten the account url helper for free. Just fyi

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,5 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+
+10.times { Account.create }
+10.times { User.create }
```
Comment (Andrew-Max) [inline db/seeds.rb:5]: Do you understand what this file is doing?

File: app/views/accounts/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+this is the show page for <%=@account.id%>
+
```
Comment (Andrew-Max) [inline app/views/accounts/show.html.erb:2]: this is my bad but the spacing on `<%=@account.id%>` should be `<%= @account.id %>`
pr is the time when you should be catching this kinda stuff though.

File: app/views/accounts/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<% @accounts.each do |account| %>
+  <%= link_to account.id, account_path(id: account.id) %>
+<% end %>
+
```
Comment (Andrew-Max) [inline app/views/accounts/index.html.erb]: Wrap this whole thing in a `<ul>` tag and then place each `link_to` in a `<li>` tag so this renders as a list as opposed to all smashed together.

File: db/seeds.rb (hunk 2 above)
Comment (jkranking) [reply db/seeds.rb:5]: yeah. setting up values in the database.

File: app/views/accounts/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<% @accounts.each do |account| %>
+  <%= link_to account.id, account_path(id: account.id) %>
+
```
Comment (Andrew-Max) [inline app/views/accounts/index.html.erb]: wrap this in ul and li

File: app/views/accounts/show.html.erb (hunk 3 above)
Comment (Andrew-Max) [reply app/views/accounts/show.html.erb:2]: I fixed this

File: app/views/accounts/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<ul>
+<% @accounts.each do |account| %>
+  <li><%= link_to account.id, account_path(id: account.id) %></li>
+<% end %>
+
+</ul>
```
Comment (Andrew-Max) [inline app/views/accounts/index.html.erb]: needs end line

File: app/views/accounts/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<ul>
+<% @accounts.each do |account| %>
```
Comment (Andrew-Max) [inline app/views/accounts/index.html.erb]: needs another indent after ul

File: app/views/accounts/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+  <li><%= link_to account.id, account_path(id: account.id) %></li>
+<% end %>
+
```
Comment (Andrew-Max) [inline app/views/accounts/index.html.erb]: no blank line here

File: app/views/accounts/index.html.erb [hunk 9]
```diff
@@ -0,0 +1,5 @@
+<ul>
+  <% @accounts.each do |account| %>
+  <li><%= link_to account.id, account_path(id: account.id) %></li>
+<% end %>
+</ul>
```
Comment (Andrew-Max) [inline app/views/accounts/index.html.erb:7]:
```
<ul>
  <% @accounts.each do |account| %>
    <li>
       <%= link_to account.id, account_path(id: account.id) %>
     </li>
   <% end %>
 </ul>
```

**Review by Andrew-Max — APPROVED** [review]

**Pull request merged**